package.json

```diff
@@ -32,6 +32,7 @@
   },
   "devDependencies": {
     "broccoli-asset-rev": "^2.4.5",
+    "bulma": "^0.1.2",
```

Comment:
@jbailey4 Yeah, that could be an issue for some, but is also configurable. I would say we just cut a new release and add that to the release notes specifically. What do you think @alexdiliberto ?